server/utils/placementRequests/adminSummary.ts

```diff
@@ -0,0 +1,78 @@
+import { PlacementRequestDetail } from '../../@types/shared'
+import { SummaryList } from '../../@types/ui'
+import { allApTypes } from '../allAPTypes'
+import { allReleaseTypes } from '../applications/releaseTypeUtils'
+import { DateFormats } from '../dateUtils'
+import { placementDates, placementLength } from '../matchUtils'
+
+export const adminSummary = (placementRequest: PlacementRequestDetail): SummaryList => {
+  const dates = placementDates(placementRequest.expectedArrival, String(placementRequest.duration))
+  return {
+    rows: [
+      {
+        key: {
+          text: 'CRN',
+        },
+        value: {
+          text: placementRequest.person.crn,
+        },
+      },
+      {
+        key: {
+          text: 'Tier',
+        },
+        value: {
+          text: placementRequest.risks.tier.value.level,
```

Comment:
We should handle the cases where `tier.value` is undefined (strict mode would've caught this)